example/timer/odp_timer_accuracy.c

```diff
@@ -219,6 +216,8 @@ static int start_timers(test_global_t *test_global)
 		test_global->timer[i] = timer;
 	}
 
+	start_tick = 0;
```

Comment:
not needed.

Reply:
needed by some compiler versions (GCC for ARM) which think that it may be used uninitialized otherwise.

Reply:
ok if compiler warns.

Reply:
GCC has an unfortunate habit of not being too good at seeing whether variables are really uninitialized. The Linux kernel typically flags these superfluous initializations with a comment so that some future maintenance doesn't delete them without further investigation. Might be a good practice to adopt here.

Reply:
Currently, our build fails on Travis if this line is removed (that's why I needed to add it). So, if someone removes it later Travis will tell if it's OK, or not. It's OK to remove it when upgraded compilers do not need it anymore.